lib/adapter.js

```diff
@@ -214,10 +214,13 @@ function start(options) {
     // FIXME confirm appmetrics.configure() accepts sl-run:agentOptions
     appmetrics.enable('eventloop');
     monitor = appmetrics.monitor();
+
+    global.APPMETRICS = appmetrics;
+    global.APPMETRICS_MONITOR = monitor;
+
     initialise();
     appmetricsStarted = true;
 
-    global.APPMETRICS = appmetrics;
```

Comment:
I thought the `initialise()` function call would spin up the watchers before that definition was set?